README.md

```diff
@@ -1 +1,41 @@
-# TeamCity
\ No newline at end of file
+# Leapwork-Integration
+This is Leapwork plugin for Teamcity
+
+# More Details
+Leapwork is a mighty automation testing system and now it can be used for running [smoke, functional, acceptance] tests, generating reports and a lot more in Teamcity. You can easily configure integration directly in Teamcity enjoying UI friendly configuration page with easy connection and test suites selection.
+
+# Features:
+ - Setup and test Leapwork connection in few clicks
+ - Run automated tests in your Teamcity build tasks
+ - Automatically receive test results
+ - Build status based tests results
+ - Write tests trace to build output log
+ - Smart UI
+ 
+# Installing
+- Use maven.
+- Command: mvn package 
+- Or simply install zip-file from the "target" folder: Copy the zip plugin package into the {TeamCity Data Directory}/plugins directory (Default path: C:\ProgramData\JetBrains\TeamCity\plugins). 
+- If you have an earlier version of the plugin in the directory, remove it.
+- Alternatively, use the Administration -> Plugins List -> Upload plugin zip -> Choose File -> Choose that zip-file -> Press Upload plugin zip
+
+# Update 4.0.1
+- For Leapwork version 2021.1
+- Fixed bug when schedules are executing in non-ordered way.
```

Comment:
Replace all update section to
# Update 4.0.1
- Leapwork version 2021.1 is now supported
- Plugin is now using Leapwork API v4